draft-ietf-rats-architecture.md

```diff
@@ -127,6 +127,100 @@ Appraisal Policy to make application-specific decisions such as authorization de
+So, in this case, the multi-chassis router is the Composite Attester,
+each router is an Attester and the main router is the Lead Attester.
+
+{{composite}} depicts the data that flows between the Composite Attester and Verifier for the remote attestation.
+
+~~~~
+                   .-----------------------------.
+                   |           Verifier          |
+                   '-----------------------------'
+                                 |  ^
+           Endorsements and      |  | Evidence of
+           Appraisal Policy for  |  | Composite
+           Evidence of Attesters |  | Attester
+                                 |  |
+.--------------------------------|--|-------------------------------.
+|                                v  |                               |
+|  .--------------------------------------.                         |
+|  |                   .-------------.    |  Evidence of Attesters  |
+|  |                   | Attesting   |-.  | (via Internal Links or  |
+|  | Collecting +------| Environment | |  |  Network Connections)   |
+|  | Claims     |      '-------------' |  |    /                    |
+|  |            |        '-------------'  |   / .------------.      |
+|  |            v                         |  /  |            |      |
+|  |  .-------------.    ***************  |<----| Attester B |-.    |
+|  |  | Target      |-.  * Verifying   *  |     '------------' |    |
+|  |  | Environment | |  * Environment *  |<------| Attester C |-.  |
+|  |  '-------------' |  * (Optional)  *  |       '------------' |  |
+|  |    '-------------'  ***************  |<--------| ......     |  |
+|  | Lead Attester A                      |         '------------'  |
+|  '--------------------------------------'                         |
+|                                                                   |
+|                       Composite Attester                          |
+'-------------------------------------------------------------------'
+~~~~
+{: #composite title="Conceptual Data Flow for Composite Attester"}
+
+In the Composite Attester, each Attester generates its own Evidence by its
+Attesting Environments collecting the claims from its Target Environments.
+The Lead Attester collects the Evidence of all other Attesters and then
+generates the Evidence of the whole Composite Attester.
+
+Inside the Lead Attester, there may be an optional Verifying Environment.
+The Verifying Environment can verify the collected Evidence of other
+Attesters to evaluate their trustworthiness. Therefore, there are two
+situations when the Lead Attester generates the final Evidence.
+
+One situation is that the Lead Attester has no Verifying Environment.
+In this situation, the Lead Attester just collects the Evidence of other
+Attesters but doesn't verify them. It may just string all these Evidence
+into a whole one, or it may reorganize these Evidence with a new
+structure and sign this final Evidence. Then it conveys the final Evidence
```

Comment:
Use/introduce Composite Evidence term as the structure to sign this final evidence.